### DISCUSSION TITLE
Where is the cicd pipeline for pyarmor?

### DISCUSSION THREAD

**Discussion by eshafaq1** [dashingsoft/pyarmor#1826] [Q&A]
@jondy Greetings, Is it possible to get some more information on how the pip module is packaged? Would like some knowledge and understanding on pyarmor CI pipeline and release process after a release is tagged.

**Comment by jondy** [comment]
Maybe this would help a little
https://pyarmor.readthedocs.io/en/latest/how-to/ci.html
